File: 2073-time-needed-to-buy-tickets/2073-time-needed-to-buy-tickets.java
class Solution {
    public int timeRequiredToBuy(int[] tickets, int k) {
        int ans=0;
        for(int i=0;i<tickets.length;i++)
        {
            if(k<i)ans=ans+Math.min(tickets[k]-1,tickets[i]);
            else ans=ans+Math.min(tickets[k],tickets[i]);
        }
        return ans;
    }
}

File: 2073-time-needed-to-buy-tickets/2073-time-needed-to-buy-tickets.cpp
class Solution {
public:
    int timeRequiredToBuy(vector<int>& tickets, int k) {
        int ans=0;
        for(int i=0;i<tickets.size();i++) {
            if(i>k)ans=ans+min(tickets[k]-1,tickets[i]);
            else ans=ans+min(tickets[k],tickets[i]);
        }
        return ans;
    }
};